src/main.zig

```diff
@@ -15,7 +15,6 @@ const should_enable_signal_handling = !is_windows and builtin.os.tag != .wasi;
 
 const logger = std.log.scoped(.zigline);
 
-// std.os is a LIE!
```

Comment:
It no longer is :)

Reply:
o rly?

Reply:
https://github.com/ziglang/zig/pull/19354

Reply:
I like where this is headed :eyes: